src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -230,6 +231,15 @@ public boolean isEmpty() {
     }
 
     /**
+     * Returns the sequence dictionaries associated with the given set of feature sources.
+     */
+    public List<SAMSequenceDictionary> getAllSequenceDictionaries() {
+        return featureSources.values().stream().map(fs -> fs.getSequenceDictionary())
+                .filter(dict -> dict != null)
+                .collect(Collectors.toList());
+    }
```

Comment:
Also, document the fact that the method will silently return an empty list of none of the feature sources have dictionaries.